api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TXTBCohortQueries.java

```diff
@@ -828,4 +840,126 @@ public CohortDefinition txTbDenominator() {
     addGeneralParameters(cd);
     return cd;
   }
+
+  public CohortDefinition positiveScreening() {
+    CompositionCohortDefinition cd = new CompositionCohortDefinition();
+    CohortDefinition A = codedYesTbScreening();
+    addGeneralParameters(A);
+    cd.addSearch("A", map(A, codedObsParameterMapping));
+    CohortDefinition B = positiveOrNegativeInvesitionResult();
+    addGeneralParameters(B);
+    cd.addSearch("B", map(B, generalParameterMapping));
+    CohortDefinition C = tbTreatmentStartDateWithinReportingDate();
+    addGeneralParameters(C);
+    cd.addSearch("C", map(C, generalParameterMapping));
+    CohortDefinition D = getInTBProgram();
+    addGeneralParameters(D);
+    cd.addSearch("D", map(D, generalParameterMapping));
+    cd.setCompositionString("A OR B OR C OR D");
+    addGeneralParameters(cd);
+    return cd;
+  }
+
+  public CohortDefinition negativeScreening() {
+    CompositionCohortDefinition cd = new CompositionCohortDefinition();
+    CohortDefinition A = tbScreening();
+    addGeneralParameters(A);
+    cd.addSearch("A", map(A, codedObsParameterMapping));
+    CohortDefinition B = positiveScreening();
+    addGeneralParameters(B);
+    cd.addSearch("B", map(B, generalParameterMapping));
+    cd.setCompositionString("A NOT B");
+    addGeneralParameters(cd);
+    return cd;
+  }
+
+  public CohortDefinition newOnARTPositiveScreening() {
+    CompositionCohortDefinition cd = new CompositionCohortDefinition();
+    CohortDefinition DEN = txTbDenominator();
+    addGeneralParameters(DEN);
+    cd.addSearch("DEN", map(DEN, generalParameterMapping));
+    CohortDefinition A = artListA();
+    addGeneralParameters(A);
+    cd.addSearch("A", map(A, generalParameterMapping));
+    CohortDefinition B = positiveScreening();
+    addGeneralParameters(B);
+    cd.addSearch("B", map(B, generalParameterMapping));
+    cd.setCompositionString("DEN INTERSECTION A INTERSECTION B");
```

Comment:
Is this line suppossed to be **DEN AND A AND B** I haven't used INTERSECTION before, I thought INTERSECTION means **AND**